config/sync/core.entity_form_display.paragraph.components.default.yml

```diff
@@ -24,7 +24,7 @@ content:
       closed_mode_threshold: 0
       add_mode: dropdown
       form_display_mode: default
-      default_paragraph_type: _none
+      default_paragraph_type: rich_text
```

Comment:
Does this work for you?

Reply:
Amended, as discussed.

Reply:
Thanks but i think i made a mistake, change the default paragraph back to none please, i think it's better just to have the drop button have rich text by default instead of always forcing the the user to add in rich text or have to have to remove it.

Reply:
that's alright. Done.